src/Controller/FacebookController.php

```diff
@@ -91,7 +96,7 @@ public function logUserIn(Request $request, PDO $db)
         }
 
         trigger_error('Unexpected Facebook error (' . $res->getStatusCode()
-            . ': ' . $res->getBody() . ')', E_USER_WARNING);
+                      . ': ' . $res->getBody() . ')', E_USER_WARNING);
```

Comment:
This block is really difficult to read. Can we clean it up using something like:

```php
trigger_error(
    sprintf(
        'Unexpected Facebook error (%s: %s)',
        $res->getStatusCode(),
        $res->getBody()
    ),
    E_USER_WARNING
);
```

Reply:
changed in latest commits